src/main/java/com/entropyteam/entropay/employees/repositories/PtoRepository.java

```diff
@@ -14,4 +16,20 @@ public interface PtoRepository extends BaseRepository<Pto, UUID> {
             "ORDER BY ABS(EXTRACT(EPOCH FROM (start_date - CURRENT_TIMESTAMP))) " +
             "LIMIT 1", nativeQuery = true)
     LocalDate findNearestPto(@Param("employeeId") UUID employeeId);
+
+    @Query("SELECT p FROM Pto p " +
+            "WHERE p.deleted = false " +
+            "AND p.status = 'APPROVED'")
+    List<Pto> findAllByDeletedIsFalseAndStatusIsApproved();
+
+    @Query(value = "SELECT * FROM pto " +
+            "WHERE deleted = false " +
+            "AND status = 'APPROVED' " +
+            "AND EXTRACT(YEAR FROM start_date) = :year " +
+            "OR EXTRACT(YEAR FROM end_date) = :year", nativeQuery = true)
+    List<Pto> findAllByDeletedIsFalseAndStatusIsApprovedForYear(@Param("year") int year);
+
+    @Query(value = "SELECT DISTINCT extract('Year' FROM start_date) AS year FROM pto WHERE deleted=false "
```

Comment:
Minor: change format from `extract('Year' FROM start_date)` to `EXTRACT(YEAR FROM start_date)`